File: BIPClient/BIPFramework/core/ReturnInfo.cs
using System;

namespace com.ccf.bip.framework.core
{
    /// <summary>
    /// 平台服务调用统一返回信息
    /// </summary>
    [Serializable]
    public class ReturnInfo
    {
        private int code;
        /// <summary>
        /// 错误码 0-正确 <>0错误(错误码未定义)
        /// </summary>
        public int Code
        {
            get { return code; }
            set { code = value; }
        }

        private String message;
        /// <summary>
        /// 错误描述
        /// </summary>
        public String Message
        {
            get { return message; }
            set { message = value; }
        }

        private Object value;
        /// <summary>
        /// 返回值
        /// </summary>
        public Object Value
        {
            get { return this.value; }
            set { this.value = value; }
        }
    }
}

File: BIPServer/src/com/ccf/bip/framework/core/ReturnInfo.java
package com.ccf.bip.framework.core;

import java.io.Serializable;

public class ReturnInfo implements Serializable {
    private static final long serialVersionUID = -5450924816521698349L;
    
    private int code;
    private String message;
    private Object value;
    
    public int getCode() {
        return code;
    }
    public void setCode(int code) {
        this.code = code;
    }
    public String getMessage() {
        return message;
    }
    public void setMessage(String message) {
        this.message = message;
    }
    public Object getValue() {
        return value;
    }
    public void setValue(Object value) {
        this.value = value;
    }
}
